packages/react-saasify/src/themes/waves.module.css

```diff
@@ -0,0 +1,598 @@
+}
+
+header.container .link:hover {
+  color: var(--color-primary) !important;
+  filter: var(--filter-tint);
+}
+
+header.container .links .link {
+  margin-left: var(--space-xl);
+  margin-right: var(--space-xl);
+}
+
+header.container .links .link:first-of-type {
+  margin-left: 0;
+}
+
+header.container .links .link:nth-of-type(2),
+header.container .links .link:nth-of-type(5) {
+  display: none;
+}
+
+header.container .links .link:nth-of-type(3) {
+  order: 2;
+  opacity: 0.9;
+}
+
+header.container .links .link:nth-of-type(4) {
+  margin-left: auto;
+  margin-right: var(--space-xl);
+  order: 1;
+  opacity: 0.9;
+}
+
+header.container .actions .link:first-of-type {
+  margin-left: var(--space-xl);
+}
+
+header.container .actions .link {
+  margin-left: var(--space-m);
+}
+
+header .content {
+  max-width: var(--max-width) !important;
+}
+
+header.attached {
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.08);
+  padding-bottom: var(--space-s) !important;
+  padding-top: var(--space-s) !important;
+}
+
+/*
+ * Footer
+ * ----------------------------
+ */
+
+footer {
+  background-color: #f9f9fb !important;
+  border-top: 1px solid #efefef;
+}
+
+footer .detail,
+footer .header {
+  color: var(--color-fg) !important;
+  opacity: 0.7;
+}
+
+footer .link {
+  color: var(--color-fg) !important;
+  opacity: 0.7;
+}
+
+footer .link:hover {
+  color: var(--color-primary) !important;
+  filter: var(--filter-tint);
+  opacity: 1;
+}
+
+/*
+ * CTAs
+ * ----------------------------
+ */
+.theme-waves a {
+  color: var(--color-primary);
+}
+
+.theme-waves a:hover {
+  filter: var(--filter-tint);
+}
+
+/* TODO: cta-button ghost should have dark text */
+.cta-button,
+.cta-button.inline {
+  border: var(--color-primary) solid 1px !important;
+  font-size: 16px !important;
+  font-weight: 500;
+  letter-spacing: normal !important;
+  text-transform: none !important;
+  text-shadow: none;
+  padding-bottom: var(--space-s);
+  padding-top: var(--space-s);
+  padding-left: var(--space-xl);
+  padding-right: var(--space-xl);
+}
+
+.cta-button:global(.primary) {
+  background: var(--color-primary) !important;
+  color: white !important;
+}
+
+.cta-button:global(.secondary) {
+  background: transparent !important;
+  color: var(--color-primary) !important;
+}
+
+.cta-button:hover {
+  box-shadow: 0 7px 14px rgba(50, 50, 93, 0.11), 0 3px 6px rgba(0, 0, 0, 0.08);
+  filter: var(--filter-tint);
+}
+
+.cta-button:global(.secondary):hover {
+  color: var(--color-primary) !important;
+}
+
+/*
+ * Code Block
+ * ----------------------------
+ */
+.live-service-demo {
+  background-image: linear-gradient(174deg, #6e60e1 0%, #1b3b87 93%);
+  border-radius: 8px;
+  box-shadow: none !important;
+  filter: drop-shadow(0px 3px 6px #392ab195);
+  overflow: hidden;
+}
+
+.live-service-demo .tabs:before {
+  align-self: center;
+  color: #ffffff90;
+  content: 'Live Demo';
+  font-size: 16px;
+  font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier,
```

Comment:
are these added to the theme's fonts or just commonly installed?

Reply:
Common